tests/cpu/test_kalman_fitter_wire_chamber.cpp

```diff
@@ -0,0 +1,210 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023 CERN for the benefit of the ACTS project
+ *
+ * Mozilla Public License Version 2.0
+ */
+
+// Project include(s).
+#include "traccc/edm/track_state.hpp"
+#include "traccc/fitting/fitting_algorithm.hpp"
+#include "traccc/io/utils.hpp"
+#include "traccc/resolution/fitting_performance_writer.hpp"
+#include "traccc/simulation/measurement_smearer.hpp"
+#include "traccc/simulation/simulator.hpp"
+#include "traccc/simulation/smearing_writer.hpp"
+#include "traccc/utils/ranges.hpp"
+#include "traccc/utils/seed_generator.hpp"
+
+// Test include(s).
+#include "tests/kalman_fitting_wire_chamber_test.hpp"
+
+// detray include(s).
+#include "detray/simulation/event_generator/track_generators.hpp"
+
+// VecMem include(s).
+#include <vecmem/memory/host_memory_resource.hpp>
+
+// GTest include(s).
+#include <gtest/gtest.h>
+
+// System include(s).
+#include <filesystem>
+#include <string>
+
+using namespace traccc;
+
+TEST_P(KalmanFittingWireChamberTests, Run) {
+
+    // Get the parameters
+    const std::string name = std::get<0>(GetParam());
+    const std::array<scalar, 3u> origin = std::get<1>(GetParam());
+    const std::array<scalar, 3u> origin_stddev = std::get<2>(GetParam());
+    const std::array<scalar, 2u> mom_range = std::get<3>(GetParam());
+    const std::array<scalar, 2u> eta_range = std::get<4>(GetParam());
+    const std::array<scalar, 2u> theta_range = eta_to_theta_range(eta_range);
+    const std::array<scalar, 2u> phi_range = std::get<5>(GetParam());
+    const unsigned int n_truth_tracks = std::get<6>(GetParam());
+    const unsigned int n_events = std::get<7>(GetParam());
+
+    // Performance writer
+    traccc::fitting_performance_writer::config fit_writer_cfg;
+    fit_writer_cfg.file_path = "performance_track_fitting_" + name + ".root";
+    traccc::fitting_performance_writer fit_performance_writer(fit_writer_cfg);
+
+    /*****************************
+     * Build a drift chamber
+     *****************************/
+    // Memory resources used by the application.
+    vecmem::host_memory_resource host_mr;
+
+    // Read back detector file
+    detray::io::detector_reader_config reader_cfg{};
+    reader_cfg.add_file("wire_chamber_geometry.json")
+        .add_file("wire_chamber_homogeneous_material.json")
+        //.add_file("wire_chamber_surface_grids.json")
```

Comment:
This line is causing error only for traccc, which is very odd.
After discussing with @niermann999, we decided to leave it commented for the moment